numpy/lib/tests/test_io.py

```diff
@@ -1878,49 +1878,52 @@ def test_genfromtxt(self):
             np.savetxt(path, a)
             data = np.genfromtxt(path)
             assert_array_equal(a, data)
-    
+
     @np.testing.dec.skipif(Path is None, "No pathlib.Path")
     def test_ndfromtxt(self):
         # Test outputing a standard ndarray
         with temppath(suffix='.txt') as path:
             path = Path(path)
             with path.open('w') as f:
-                f.write('1 2\n3 4')
+                f.write(u'1 2\n3 4')
+
             control = np.array([[1, 2], [3, 4]], dtype=int)
             test = np.ndfromtxt(path, dtype=int)
             assert_array_equal(test, control)
-    
+
     @np.testing.dec.skipif(Path is None, "No pathlib.Path")
     def test_mafromtxt(self):
         # From `test_fancy_dtype_alt` above
         with temppath(suffix='.txt') as path:
             path = Path(path)
             with path.open('w') as f:
-                f.write('1,2,3.0\n4,5,6.0\n')
-                
+                f.write(u'1,2,3.0\n4,5,6.0\n')
```

Comment:
See above. "Official" Python pathlib is new in 3.4 and as I understand it, remains somewhat controversial. I wonder if this requirement might be considered a bug.